### PR TITLE
Add png output to Rmd scripts

### PR DESCRIPTION
**Pull request by mfebrizio**
Nate suggested that having image files to put on the main Reg Stats page, in addition to the PDFs, would be helpful. This code implements that suggestion.

### CODE REVIEW

File: charts/code/agency_econ_significant_rules_by_presidential_year.Rmd
Comment (mfebrizio) [inline charts/code/agency_econ_significant_rules_by_presidential_year.Rmd:1]: @pateldevenr can you review these revisions? in particular, see how I looped over a vector of agency acronyms to get the code to handle multiple agencies. first, I think it doesn't change the output, but you may want to ensure that. second, would you be able to implement a similar loop for the entirety of this Rmd? I see you've already made functions for graphing, etc., so much of the work is already done.

**Comment by mfebrizio** [comment]
bumping this for review -- @zhoudanxie, @haysarah, and @yqz5514, it would be great if you could take a look by this Wednesday

File: charts/code/agency_econ_significant_rules_by_presidential_year.Rmd
Comment (pateldevenr) [reply charts/code/agency_econ_significant_rules_by_presidential_year.Rmd:1]: I've checked that the loop does not change the output so I think we are good there.  I will continue to work on converting this entire Rmd to a loop.

Comment (mfebrizio) [reply charts/code/agency_econ_significant_rules_by_presidential_year.Rmd:1]: Awesome, thanks Deven. Please "approve" the PR when you get a chance

**Review by pateldevenr — APPROVED** [review]
addition of the PNG and PDF loops looks good on the agency charts!

**Review by zhoudanxie — APPROVED** [review]
I tested the R code for all the charts except the agency ones, and it worked well. The output PNGs look good.

**Pull request merged**